daily-challenges/2024-09/18/hunghoang3110.spec.ts

```
@@ -0,0 +1,53 @@
+        age = curYear - input;
+    }
+    return `Your age: ${age}`
+}
+
+// Playwright
+import {test, expect} from '@playwright/test'
+
+test('fill in to do', async ({page}) => {
+    
+    // Prepate data
+    let firstContent = 'Xin chào, đây là bài thực hành ngày 18 tháng 9'
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'firstContent' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)